### PR TITLE
Update dino.icu.yaml

### PR DESCRIPTION
**Pull request by leecheeyong**
Requesting this domain for an event, thanks

### CODE REVIEW

File: dino.icu.yaml [hunk 1]
```diff
@@ -66,7 +66,12 @@ art: # by https://github.com/rajanwastaken
   ttl: 1
   type: CNAME
   value: particle-generativeart.netlify.app.
-
+  
+canteen:
+  ttl: 1
+  type: A
+  value: 76.76.21.21 # vercel
+
```
Comment (Muirrum) [inline dino.icu.yaml:74]: If this is vercel, this should be a CNAME to `cname.vercel-dns.com` with a ttl of 600.

**Review by Muirrum — CHANGES_REQUESTED** [review]
You should get the verification record and submit it as part of this PR

File: dino.icu.yaml (hunk 1 above)
Comment (leecheeyong) [reply dino.icu.yaml:74]: I'm unsure if this will work as there is only an option for A record but I'll give it a try

Comment (leecheeyong) [reply dino.icu.yaml]: image

i don't see any verification step

**Comment by leecheeyong** [comment]
I do not have any verification record, this is what vercel asked for although I've made changes according to the example you gave 

image

**Pull request merged**